official/vision/beta/projects/yolo/configs/experiments/darknet53.yaml

```diff
@@ -0,0 +1,53 @@
+runtime:
+  distribution_strategy: 'mirrored'
+  mixed_precision_dtype: 'float32'
+  loss_scale: 'dynamic'
+task:
+  model:
+    num_classes: 1001
+    input_size: [224, 224, 3]
+    backbone:
+      type: 'darknet'
+      darknet:
+        model_id: 'darknet53'
+  losses:
+    l2_weight_decay: 0.0005
+    one_hot: True
+  train_data:
+    tfds_name: 'imagenet_a'
+    tfds_split: 'test'
+    tfds_download: True
+    is_training: True
+    global_batch_size: 2
+    dtype: 'float16'
+  validation_data:
+    tfds_name: 'imagenet_a'
+    tfds_split: 'test'
+    tfds_download: True
+    is_training: False
+    global_batch_size: 2
+    dtype: 'float16'
+    drop_remainder: False
+trainer:
+  train_steps: 51200000 # in the paper
+  validation_steps: 25600 # size of validation data
+  validation_interval: 150 
+  steps_per_loop: 150
+  summary_interval: 150
+  checkpoint_interval: 150
+  optimizer_config:
+    optimizer:
+      type: 'sgd'
+      sgd:
+        momentum: 0.9
+    learning_rate:
+      type: 'polynomial'
+      polynomial:
+        initial_learning_rate: 0.1
+        end_learning_rate: 0.0001
+        power: 4.0 
+        decay_steps: 51136000
+    warmup:
+      type: 'linear'
+      linear:
+        warmup_steps: 64000 #lr rise from 0 to 0.1 over 1000 steps
```

Comment:
Not sure of the notation, but perhaps this should say not "#lr" but "# linear" ?